### DISCUSSION TITLE
Is it possible to change the location in which vim-fugitive tabs appear?

### DISCUSSION THREAD

**Discussion by mbomfim33** [tpope/vim-fugitive#1729] [Q&A] [answered]
I've just recently entered the vim world, and I'm wondering if someone has some ideas as to how I can change the way vim-fugitive Panel opens up. At the moment, it just opens to the left, but when using NerdTree, for instance, it ends up cluttering the screen: 
![image](https://user-images.githubusercontent.com/30595009/114605447-7baae680-9c70-11eb-9f49-9a8995400854.png)

Let me know if I should bring this up somewhere else!

**Comment by tpope** [comment, marked as answer]
You can `:Git! blame` to get a horizontal split, but given the way the two windows are scrollbound I wouldn't recommend it for most use cases.
